reference/reference.go

```diff
@@ -168,6 +169,20 @@ func ParseIDOrReference(idOrRef string) (digest.Digest, Named, error) {
 	return "", ref, err
 }
 
+// Match reports whether ref matches the specified pattern
+func Match(pattern string, ref Named) bool {
+	// FIXME: should we support pattern for NamedTagged or not ?
```

Comment:
The lines below are to keep the current behaviour, not sure I like that… We might want to implement supporting `foo/**/[ai]:ta*` and keep the current behavior in `images.go` 👼 @stevvooe wdyt ?